common/common.go

```diff
@@ -102,6 +102,11 @@ func PodManifestPath(root string) string {
 	return filepath.Join(root, "pod")
 }
 
+// PodManifestLockPath returns the path in root to the Pod Manifest lock file
+func PodManifestLockPath(root string) string {
```

Comment:
right, I think I asked this a while back but I've forgotten the answer so let's have it written down :-)